src/explore-education-statistics-admin/src/pages/release/pre-release/PreReleaseMethodologiesPage.tsx

```diff
@@ -0,0 +1,98 @@
+import Link from '@admin/components/Link';
+import PageTitle from '@admin/components/PageTitle';
+import {
+  PreReleaseMethodologyRouteParams,
+  preReleaseMethodologyRoute,
+} from '@admin/routes/preReleaseRoutes';
+import { ReleaseRouteParams } from '@admin/routes/releaseRoutes';
+import publicationService from '@admin/services/publicationService';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import Tag from '@common/components/Tag';
+import TagGroup from '@common/components/TagGroup';
+import WarningMessage from '@common/components/WarningMessage';
+import useAsyncHandledRetry from '@common/hooks/useAsyncHandledRetry';
+import React from 'react';
+import { generatePath, RouteComponentProps } from 'react-router';
+
+const PreReleaseMethodologiesPage = ({
+  match,
+}: RouteComponentProps<ReleaseRouteParams>) => {
+  const { publicationId, releaseId } = match.params;
+
+  const { value: publication, isLoading } = useAsyncHandledRetry(
+    () => publicationService.getMyPublication(publicationId),
+    [publicationId],
+  );
+
+  const methodologies = publication?.methodologies.filter(
+    ({ methodology }) =>
+      methodology.status !== 'Draft' ||
+      (methodology.status === 'Draft' && methodology.amendment),
+  );
+
+  return (
+    <div className="govuk-width-container">
+      <>
+        <PageTitle title="Methodologies" />
+        <LoadingSpinner loading={isLoading}>
+          {publication && (
+            <>
+              {methodologies?.length === 0 &&
+              !publication.externalMethodology ? (
+                <WarningMessage>No Methodologies available.</WarningMessage>
```

Comment:
Think it would be a lowercase 'm' here, in line with existing messages like "No releases available."

Reply:
Yeah good point, that's changed here and in the tests now.